<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 728, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 834, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 894, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_syntax.py", line 41, in check_recipe
    spec = cast(str, ro_parser.get_value(path))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/parser/recipe_parser.py", line 983, in get_value
    raise KeyError(f"No value/key found at path {path!r}")
KeyError: "No value/key found at path '/requirements/host/0'"

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_1eppl_8vcf/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -12,97 +12,107 @@ source:
 
 build:
   number: 0
+  missing_dso_whitelist:     # [s390x]
+    - "$RPATH/ld64.so.1"     # [s390x]
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
     - make                   # [unix]
     - wget                   # [unix]
     - unzip                  # [unix]
+    - m2-patch               # [win]
+    - patch                  # [not win]
   host:
   run:
 
 outputs:
   - name: libmpdec
     files:
-      - lib/libmpdec.so.*
-      - lib/libmpdec.*.dylib
-      - Library/bin/libmpdec-*.dll
+      - lib/libmpdec.so.*           # [linux]
+      - lib/libmpdec.*.dylib        # [osx]
+      - Library/bin/libmpdec-*.dll  # [win]
     requirements:
       build:
         - {{ compiler('c') }}
-        - {{ stdlib('c') }}
     test:
       commands:
-        - test -f ${PREFIX}/lib/libmpdec.so.4     # [linux]
-        - test -f ${PREFIX}/lib/libmpdec.4.dylib  # [osx]
-        - if not exist %LIBRARY_BIN%\libmpdec-4.dll exit 1   # [win]
+        - test -f ${PREFIX}/lib/libmpdec.so.{{ version[0] }}               # [linux]
+        - test -f ${PREFIX}/lib/libmpdec.so.{{ version }}                  # [linux]
+        - test -f ${PREFIX}/lib/libmpdec.{{ version[0] }}.dylib            # [osx]
+        - if not exist %LIBRARY_BIN%\libmpdec-{{ version[0] }}.dll exit 1  # [win]
 
   - name: libmpdecxx
     files:
-      - lib/libmpdec++.so.*
-      - lib/libmpdec++.*.dylib
-      - Library/bin/libmpdec++-*.dll
+      - lib/libmpdec++.so.*           # [linux]
+      - lib/libmpdec++.*.dylib        # [osx]
+      - Library/bin/libmpdec++-*.dll  # [win]
     requirements:
       build:
         - {{ compiler('cxx') }}
-        - {{ stdlib('c') }}
       host:
         - {{ pin_subpackage("libmpdec", exact=True) }}
       run:
         - {{ pin_subpackage("libmpdec", exact=True) }}
     test:
       commands:
-        - test -f ${PREFIX}/lib/libmpdec++.so.4     # [linux]
-        - test -f ${PREFIX}/lib/libmpdec++.4.dylib  # [osx]
-        - if not exist %LIBRARY_BIN%\libmpdec++-4.dll exit 1   # [win]
+        - test -f ${PREFIX}/lib/libmpdec++.so.{{ version[0] }}               # [linux]
+        - test -f ${PREFIX}/lib/libmpdec++.so.{{ version }}                  # [linux]
+        - test -f ${PREFIX}/lib/libmpdec++.{{ version[0] }}.dylib            # [osx]
+        - if not exist %LIBRARY_BIN%\libmpdec++-{{ version[0] }}.dll exit 1  # [win]
 
   - name: libmpdec-devel
     build:
       run_exports:
         - {{ pin_subpackage("libmpdec") }}
     files:
-      - lib/libmpdec.so
-      - lib/libmpdec.dylib
-      - lib/pkgconfig/libmpdec.pc
-      - include/mpdecimal.h
-      - Library/lib/mpdec.lib
-      - Library/include/mpdecimal.h
+      - lib/libmpdec.so              # [linux]
+      - lib/libmpdec.dylib           # [osx]
+      - lib/pkgconfig/libmpdec.pc    # [not win]
+      - include/mpdecimal.h          # [not win]
+      - Library/lib/mpdec.lib        # [win]
+      - Library/include/mpdecimal.h  # [win]
     requirements:
       - {{ pin_subpackage("libmpdec", exact=True) }}
     test:
       commands:
-        - test -f ${PREFIX}/lib/libmpdec.so     # [linux]
-        - test -f ${PREFIX}/lib/libmpdec.dylib  # [osx]
-        - if not exist %LIBRARY_LIB%\mpdec.lib exit 1   # [win]
+        - test -f ${PREFIX}/lib/libmpdec.so            # [linux]
+        - test -f ${PREFIX}/lib/libmpdec.dylib         # [osx]
+        - if not exist %LIBRARY_LIB%\mpdec.lib exit 1  # [win]
 
   - name: libmpdecxx-devel
     build:
       run_exports:
         - {{ pin_subpackage("libmpdecxx") }}
     files:
-      - lib/libmpdec++.so
-      - lib/libmpdec++.dylib
-      - lib/pkgconfig/libmpdec++.pc
-      - include/decimal.hh
-      - Library/lib/mpdec++.lib
-      - Library/include/decimal.hh
+      - lib/libmpdec++.so            # [linux]
+      - lib/libmpdec++.dylib         # [osx]
+      - lib/pkgconfig/libmpdec++.pc  # [not win]
+      - include/decimal.hh           # [not win]
+      - Library/lib/mpdec++.lib      # [win]
+      - Library/include/decimal.hh   # [win]
     requirements:
       - {{ pin_subpackage("libmpdecxx", exact=True) }}
       - {{ pin_subpackage("libmpdec-devel", exact=True) }}
     test:
       commands:
-        - test -f ${PREFIX}/lib/libmpdec++.so     # [linux]
-        - test -f ${PREFIX}/lib/libmpdec++.dylib  # [osx]
-        - if not exist %LIBRARY_LIB%\mpdec++.lib exit 1   # [win]
+        - test -f ${PREFIX}/lib/libmpdec++.so            # [linux]
+        - test -f ${PREFIX}/lib/libmpdec++.dylib         # [osx]
+        - if not exist %LIBRARY_LIB%\mpdec++.lib exit 1  # [win]
 
 about:
   home: https://www.bytereef.org/mpdecimal/index.html
-  summary: A package for correctly-rounded arbitrary precision decimal floating point arithmetic
   license: BSD-2-Clause
   license_file: COPYRIGHT.txt
+  license_family: BSD
+  summary: A package for correctly-rounded arbitrary precision decimal floating point arithmetic
+  description: |
+    libmpdec is a fast C library for correctly-rounded arbitrary precision decimal floating point 
+    arithmetic. It is a complete implementation of Mike Cowlishaw/IBM's 
+    General Decimal Arithmetic Specification.
+  doc_url: https://www.bytereef.org/mpdecimal/index.html
+  dev_url: https://www.bytereef.org/mpdecimal/download.html
 
 extra:
   recipe-maintainers:

